Bio/GenBank/__init__.py

```diff
@@ -278,13 +281,45 @@ def _loc(loc_str, expected_seq_length, strand):
     >>> _loc("AL391218.9:105173..108462", 2000000, 1)
     FeatureLocation(ExactPosition(105172), ExactPosition(108462), strand=1, ref='AL391218.9')
 
+    >>> _loc("<2644..159", 2868, 1, "circular")
+    CompoundLocation([FeatureLocation(BeforePosition(2643), ExactPosition(2868), strand=1), FeatureLocation(ExactPosition(0), ExactPosition(159), strand=1)], 'join')
     """
     if ":" in loc_str:
         ref, loc_str = loc_str.split(":")
     else:
         ref = None
     try:
         s, e = loc_str.split("..")
+
+        # Attempt to fix features that span the origin
+        import warnings
```

Comment:
Minor, but you could move the imports down a few lines, inside the if statement (only import them when about to use them).

However, given there are now 12 warnings in the file, we might as well do the imports once at top level?